### PR TITLE
Fix #4872 auth_db package holds models

### LINKED ISSUE #4872: 20221123 auth_db classes and auth models at outer scope
Possibly be able to get rid init_model

### PR DESCRIPTION
**Pull request by robnagler**
- auth_db is a package with all models re-exported on sirepo.auth_db
- all auth_db model modules loaded dynamically (with package path)
- Renamed session to spa_session
- auth_db.create_or_upgrade is called by server and pkcli.admin_db_upgrade
- Created pkcli.admin.db_upgrade for #5174

### CODE REVIEW

**Comment by robnagler** [comment]
Requires https://github.com/radiasoft/pykern/pull/322 so checks won't pass until that's merged.

File: sirepo/auth_db/__init__.py
Comment (e-carlin) [inline sirepo/auth_db/__init__.py:59]: I think a comment here that this is never used with user defined data could be good and maybe moving _execute_raw_sql into this function so it is never used elsewhere.

File: sirepo/auth_db/__init__.py
Comment (e-carlin) [inline sirepo/auth_db/__init__.py:229]: This isn't backwards compatible. Packages on package path may not provide an auth_db sub-package.

**Review by e-carlin — APPROVED** [review]
Couple nits. LGTM. I'll let you merge since it is large and depends on pykern change.

File: sirepo/auth_db/__init__.py
Comment (robnagler) [reply sirepo/auth_db/__init__.py:229]: I was wondering whether to use package_path or not. I'm happy to remove. The failure is immediate so not a huge problem. Should I remove?

Comment (e-carlin) [reply sirepo/auth_db/__init__.py:229]: What about adding a check? If the package has it then use it. That is what the other areas of the code do in regards to package path.

File: sirepo/auth_db/__init__.py
Comment (robnagler) [reply sirepo/auth_db/__init__.py]: Thanks. I read about this, and we should just use placeholders, which seem to work fine for DDL. 57dd3717a

File: sirepo/auth_db/__init__.py
Comment (robnagler) [reply sirepo/auth_db/__init__.py:230]: This is a trickier case, because the other cases are looking for a module along package_path. This is loading all modules along package_path. #3933 is key here. If there's a syntax error in `auth_db/__init__.py`, it gets a not found, and there won't be an sirepo.auth_db module. The other issue is if the package path is being used to improve auth_db model or creating a new one. We don't handle the former case, and I don't think we can support the latter case. Therefore, I think this is a YAGNI, and I'll take it out.

Comment (e-carlin) [reply sirepo/auth_db/__init__.py:230]: Sounds good.

**Pull request merged**
